sys-process/procps/metadata.xml

```diff
@@ -6,7 +6,6 @@
 		<name>Gentoo Base System</name>
 	</maintainer>
 	<use>
-		<flag name="elogind">Use <pkg>sys-auth/elogind</pkg> for session tracking.</flag>
```

Comment:
Please just do these en-masse (one PR, many commits) rather than trickling them in.

Reply:
> I have tried doing that but I noticed it went unnoticed (ex. https://github.com/gentoo/gentoo/pull/39489) so I switched to more granular PRs.

You should instead speak to someone, not try to find workarounds..